authui/src/otpInput.ts

```diff
@@ -9,6 +10,14 @@ export class OtpInputController extends Controller {
 
   spans: HTMLElement[] = [];
 
+  get textField(): TextFieldController | null {
+    const ctr = this.application.getControllerForElementAndIdentifier(
+      this.inputTarget,
+      "text-field"
+    );
+    return ctr as TextFieldController | null;
+  }
+
```

Comment:
I think this is unnecessary. If text-field is applied correctly, it should do its job correctly. If not, there should be a bug we need to fix.